docs/desktop/flank_desktop.md

```diff
@@ -0,0 +1,44 @@
+# Flank desktop prototype
+
+## Prototype Flank options
+
+The prototype will run simple Android test on given apk and test apk with option to specify some flanks flags, as well
+as max tests shards. Those flags are the most popular one based on [mixpanel analytics](https://mixpanel.com/report/2288623/view/2835119/dashboards#id=879189)
```

Comment:
analytics will change over time. Will we continually update this to always reflect the most common mixpanel analytics?

Reply:
yes, but options chosen to create this POC based on this analytics